### DISCUSSION TITLE
How to get max value of two metrics?

### DISCUSSION THREAD

**Discussion by WystanW** [prometheus/prometheus#10601] [Q&A]
## Proposal
I have two metrics

sum(yottadb_pod_network_receive_bps{region_name!=""}) by (region_name) 
<img width="1782" alt="image" src="https://user-images.githubusercontent.com/18168353/163524588-e530dc65-147e-4bf4-b954-f7ea22cd5a8b.png">

sum(yottadb_pod_network_transmit_bps{region_name!=""}) by (region_name)
<img width="1786" alt="image" src="https://user-images.githubusercontent.com/18168353/163524601-2569a7cc-9886-497e-bfd4-5f3529dc7804.png">

How to get the max value of two metrics of each "region_name"?

**Comment by WystanW** [comment]
I know a>b or b is helpful, but how can I group max values by "region_name"?

**Reply by roidelapluie** [reply]
topk(2, sum(yottadb_pod_network_receive_bps{region_name!=""}) by (region_name))
